packages/api/src/clients/client-settings.ts

```diff
@@ -0,0 +1,26 @@
+export type ClientSettings = {
+  /**
+   * the URL to use for managing user oauth tokens.
+   * Specify this value if you are using a regional bot.
+   * For e.g., https://europe.token.botframework.com
+   * Default is https://token.botframework.com
+   */
+  readonly oauthUrl: string;
+};
+
+export const DEFAULT_CLIENT_SETTINGS: ClientSettings = {
+  oauthUrl: 'https://token.botframework.com',
+};
+
+export function mergeClientSettings(
+  clientSettings?: Partial<ClientSettings>
+): ClientSettings {
+  const env = typeof process === 'undefined' ? undefined : process.env;
```

Comment:
@Benjiiim responding here for threading, thanks for these suggestions!

- updated the endpoint, agree on matching with these docs
- removed the comment on resourceGroup().location and added in the manifest.json (these will go into official docs, just had them in this PR notes for ref)
- removed the code change to the template, let's capture this in the official docs instead